frontendUiImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/GitMacheteErrorHandler.java

```diff
@@ -25,9 +38,54 @@ public boolean submit(
       @Nullable String additionalInfo,
       Component parentComponent,
       Consumer<? super SubmittedReportInfo> consumer) {
+    String reportBodyURLPart;
+    try {
+      String bugReportTemplate = getBugTemplate();
+      for (java.util.Map.Entry<String, String> entry : getTemplateVariables(events, additionalInfo).entrySet()) {
+        bugReportTemplate = bugReportTemplate.replace("%${entry.getKey()}%", entry.getValue());
+      }
+      reportBodyURLPart = "body=" + bugReportTemplate;
+    } catch (IOException | AssertionError e) {
+      LOG.error("Error while creating bug report template", e);
+      reportBodyURLPart = "template=bug_report.md";
+    }
+
+    String finalReportBodyURLPart = reportBodyURLPart;
     GuiUtils.invokeLaterIfNeeded(() -> BrowserUtil.browse(
-        "https://github.com/VirtusLab/git-machete-intellij-plugin/issues/new?assignees=&labels=bug&template=bug_report.md"),
+        "https://github.com/VirtusLab/git-machete-intellij-plugin/issues/new?assignees=&labels=bug&" + finalReportBodyURLPart),
```

Comment:
Actually! if for whatever reason a non-URL-safe character ends up in the `body` param (e.g. `/`, present in line 2 of the template)... are they going to be properly URL-encoded? 🤔

Reply:
Great question. I will make sure that's the case and improve code if necessary.